helm/templates/_helpers.tpl

```diff
@@ -141,6 +141,19 @@ Get the current recommended auto instrumentation python image
 {{- printf "%s/%s:%s" .Values.manager.autoInstrumentationImage.python.repositoryDomain .Values.manager.autoInstrumentationImage.python.repository .Values.manager.autoInstrumentationImage.python.tag -}}
 {{- end -}}
 
+{{/*
```

Comment:
Done. reorganized daemonsets in respective folders.
`_helpers.tpl` is not broken to Linux and Windows since it has common information for both Linux and Windows.